src/NopeNumber.ts

```diff
@@ -2,6 +2,19 @@ import NopePrimitive from './NopePrimitive';
 import { Rule } from './types';
 
 class NopeNumber extends NopePrimitive<number> {
+  public integer(message = 'Input must be an integer'){
+    const rule: Rule<number> = entry => {
+      if (entry === undefined || entry === null) {
+        return;
+      }
+      if(!Number.isInteger(entry)){
```

Comment:
Thanks for contributing!

Could we please change this to `entry !== parseInt(entry, 10)` because IE11 doesn't support isInteger very well